### PR TITLE
Log test progress to a db if enabled

### PR DESCRIPTION
**Pull request by mponomar**

### CODE REVIEW

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: Success &#10003;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>4/521 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>queuedb_rollover_noroll1_generated<br>yast_stat4scan_generated<br>yast<br>truncatesc_ufid_on_generated</p>

File: tests/yast.test/runit [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 bash -n "$0" | exit 1
 
+. $TESTSROOTDIR/testsutil
```
Comment (riverszhang89) [inline tests/yast.test/runit:4]: testutils, not testsutil.

File: tests/yast.test/runit [hunk 2]
```diff
@@ -65,7 +67,9 @@ RC=0
 for test in `ls $FILE`;
 do
     echo "$test..."
+    testdb_note "$test start"
     tclsh $test $DB $FILEPATH $TCLCDB2 $CDB2_CONFIG
+    testdb_note "$test end"
```
Comment (riverszhang89) [inline tests/yast.test/runit:72]: This line needs to be moved after the `$?` check.

**Review by riverszhang89 — APPROVED** [review]

**Comment by riverszhang89** [comment]
go go go

**Pull request merged**